### DISCUSSION TITLE
【lecture5】学习材料及建议

### DISCUSSION THREAD

**Discussion by readygo67** [zkp-co-learning/zkp-co-learn#126] [Materials]
1. 如果有比较充足的时间, 建议先看郭宇老师plonk 的中文讲解视频
### 郭宇老师分享系列:

[[郭宇老师分享：理解 PLONK 原理（一）](https://www.notion.so/PLONK-07ff3c723c82438c83b142051d16d379?pvs=21)](https://www.notion.so/PLONK-07ff3c723c82438c83b142051d16d379?pvs=21) [https://www.notion.so/PLONK-07ff3c723c82438c83b142051d16d379?pvs=4](https://www.notion.so/PLONK-07ff3c723c82438c83b142051d16d379?pvs=21) 
[[郭老师分享：PLONK 原理（二）](https://www.notion.so/PLONK-4e9e5b94efd54bef9d9aeeb7823ef6a1?pvs=21)](https://www.notion.so/PLONK-4e9e5b94efd54bef9d9aeeb7823ef6a1?pvs=21) [https://www.notion.so/PLONK-4e9e5b94efd54bef9d9aeeb7823ef6a1?pvs=4](https://www.notion.so/PLONK-4e9e5b94efd54bef9d9aeeb7823ef6a1?pvs=21)

[[郭宇老师分享：Plonk 原理（三）Lookup argument ](https://www.notion.so/Plonk-Lookup-argument-89aedd901a39469599e43ff7016d0245?pvs=21)](https://www.notion.so/Plonk-Lookup-argument-89aedd901a39469599e43ff7016d0245?pvs=21) [https://www.notion.so/Plonk-Lookup-argument-89aedd901a39469599e43ff7016d0245?pvs=4](https://www.notion.so/Plonk-Lookup-argument-89aedd901a39469599e43ff7016d0245?pvs=21)

https://github.com/sec-bit/learning-zkp/tree/develop/plonk-intro-cn
看完上面的视频会对plonk 有一个比较感性的认识。

2. 如果时间比较紧张，可以直接看zk-learning 中的lecture5
https://zk-learning.org/assets/lecture5-2023.pdf
https://www.youtube.com/watch?v=A0oZVEXav24

3. 如果还有时间， 可以看熟悉一下plonk的python 实现:
https://github.com/0xPARC/plonkathon  
https://github.com/Antalpha-Labs/baby-plonk

4. 在本节学习中有任何问题，可以在wx中直接@keep.

**Comment by Demian101** [comment]
强烈推荐这个文字文章：https://github.com/sec-bit/learning-zkp/tree/develop/plonk-intro-cn
